lib/compare_with_wikidata.rb

```diff
@@ -52,17 +52,21 @@ def run!
         end
       end
 
-      client.edit(title: csv_page_title, text: comparison.to_csv)
+      client.edit(title: csv_page_title, text: comparison.to_csv) unless ENV.key?('DEBUG')
 
       # Apparently everything went smoothly, so overwrite the /errors
       # subpage to make sure that it's empty.
-      client.edit(title: errors_page_title, text: '')
+      client.edit(title: errors_page_title, text: '') unless ENV.key?('DEBUG')
     rescue StandardError => e
-      client.edit(title: errors_page_title, text: "<nowiki>#{e.message}</nowiki>")
+      if ENV.key?('DEBUG')
+        puts e
```

Comment:
This could perhaps be `raise e`, so that exceptions get raised normally?

Reply:
Or even just `raise`, which I think will raise the last exception when run in a `rescue` block.

Reply:
I considered doing that at the time, but thought that just printing the output was more in keeping with what else we were doing. But thinking about it a bit more, I think you're right: re-raising the error is probably cleaner, and even though we're hopefully going to get rid of this code RSN, we'll probably end up with a DEBUG mode anyway, and it would be better to have this do now what we'll want it to do then 👍